RFC-005-SWAP-Basic-Bitcoin.md

```diff
@@ -106,6 +106,62 @@ To compute the exact Bitcoin script bytes of the contract, implementations shoul
 | `refund_identity` | 74                     | 93                    | 20             | See [Parameters](#parameters)                                              |
 | `6888ac`          | 94                     | 96                    | 3              | `OP_ENDIF OP_EQUALVERIFY OP_CHECKSIG`                                      |
 
+
+Script execution overview:
+
+```
+// cur element on stack: byte
+//   evaluate byte: 01
+//   OP_IF (redeem) is entered
+//   pop byte
```

Comment:
I'm not sure if this description makes sense in that way: what does `evaluate byte: 01` mean? 
`OP_IF` pops an element from the stack and if this is != 0 it continues with the branch.